src/Resources/skeleton/crud/controller/Controller.tpl.php

```diff
@@ -74,11 +74,13 @@ public function show(<?= $entity_class_name ?> $<?= $entity_var_singular ?>): Re
     }
 
     /**
-     * @Route("/{<?= $entity_identifier ?>}/edit", name="<?= $route_name ?>_edit", methods={"GET","POST"})
+     * @Route("/{<?= $entity_identifier ?>}/edit", name="<?= $route_name ?>_edit", methods={"GET","PUT"})
      */
     public function edit(Request $request, <?= $entity_class_name ?> $<?= $entity_var_singular ?>): Response
     {
-        $form = $this->createForm(<?= $form_class_name ?>::class, $<?= $entity_var_singular ?>);
+        $form = $this->createForm(<?= $form_class_name ?>::class, $<?= $entity_var_singular ?>, array(
```

Comment:
We should use short array syntax in Symfony Style :)

Reply:
You're right, my old habit... Also because I find it more readable with array() ;)
I'll modify if PR goes further